code/hud/hudparse.cpp

```diff
@@ -3756,6 +3757,9 @@ void load_gauge_directives(gauge_settings* settings)
 	if ( optional_string("Max Line Width:") ) {
 		stuff_int(&max_line_width);
 	}
+	if (optional_string("Key Line Offset:")) {
```

Comment:
What do you think about calling this Key Line X Offset?

Reply:
Done